nuxeo-drive-client/nxdrive/osi/windows/windows.py

```diff
@@ -218,8 +290,14 @@ def unregister_contextual_menu(self):
     def register_folder_link(self, folder_path, name=None):
         file_lnk = self._get_folder_link(name)
         self._create_shortcut(file_lnk, folder_path)
+        if platform.release() == '10':
```

Comment:
Add registry entries only for windows 10.
Note: platform.release() return '10' only from Python 2.7.11
http://stackoverflow.com/questions/33431155/why-does-platform-release-return-8-in-windows-10